pkg/operator/operator.go

```diff
@@ -170,7 +172,7 @@ func New(
 		namespace:                 namespace,
 		namespaceUserWorkload:     namespaceUserWorkload,
 		client:                    c,
-		queue:                     workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "cluster-monitoring"),
+		queue:                     workqueue.NewNamedRateLimitingQueue(workqueue.NewItemExponentialFailureRateLimiter(50*time.Millisecond, 3*time.Minute), "cluster-monitoring"),
```

Comment:
this change doesn't appear to be related to the stated goals of the PR? This reads like a different back-off mechanism?
> * Update prometheus-operator first and wait for it to finish before updating the other operands concurrently.
> * Allow the reconcile loop three attempts before calling SetFailed and thus reporting available=false.

Reply:
Ah yes, I'll update the PR description. The respective [commit](https://github.com/openshift/cluster-monitoring-operator/pull/1193/commits/62e2d9dcd878389fb7d8e368e2b8727bf845da9f) has details.

Thanks, good catch.